src/components/SessionExpired.js

```diff
@@ -0,0 +1,46 @@
+import React, {useContext} from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import Card from 'components/Card';
+import ErrorMessage from 'components/Errors/ErrorMessage';
+import Link from 'components/Link';
+
+import {ConfigContext} from '../Context';
+import {flagNoActiveSubmission} from '../data/submissions';
+import useSessionTimeout from '../hooks/useSessionTimeout';
+
+const SessionExpired = () => {
+  const config = useContext(ConfigContext);
+  const [expired] = useSessionTimeout();
+
+  if (!config.debug && !expired) {
+    throw new Error("The session isn't expired");
```

Comment:
there should be some kind of marker you can use to ignore coverage for this line